core/src/com/unciv/ui/worldscreen/status/NextTurnButton.kt

```diff
@@ -33,13 +39,17 @@ class NextTurnButton(
                nextTurnAction: NextTurnAction? = null) {
         if (nextTurnAction != null) {
             this.nextTurnAction = nextTurnAction
-            setText(nextTurnAction.text.tr())
+            label.setText(nextTurnAction.text.tr())
             label.color = nextTurnAction.color
+            if (nextTurnAction.icon != null && ImageGetter.imageExists(nextTurnAction.icon))
+                iconCell.setActor(ImageGetter.getImage(nextTurnAction.icon))
```

Comment:
It doesn't look like we're resizing the image to for the text height?

Reply:
(this looks like the last change before merge 😃 )

Reply:
> It doesn't look like we're resizing the image to for the text height?

No, default icons are sized properly to align with text to serve as a reference to a modders.
But there should be a possibility to put different sizes there. I personally prefer big one in my own mods.

Reply:
I see